dev/breeze/src/airflow_breeze/utils/packages.py

```diff
@@ -950,6 +957,13 @@ def regenerate_pyproject_toml(
         formatted_cross_provider_dependencies = ""
     context["CROSS_PROVIDER_DEPENDENCIES"] = formatted_cross_provider_dependencies
     context["DEPENDENCY_GROUPS"] = formatted_dependency_groups
+    context["BUILD_SYSTEM"] = (
```

Comment:
same here -> we could use `provider.yaml`  new `build-system` flag to determine build system.